### ISSUE TITLE
TP-Link WBS210 v2: Ethernet interfaces are swapped

### ISSUE THREAD

**Issue by citronalco** [freifunk-gluon/gluon#2384]
Checked a build of the current master (https://github.com/freifunk-gluon/gluon/commit/2924afe5624d1445fdb39a6a9f2cb8470e3458d6)
The TP-Link WBS210v2 has two LAN ports, one of them, labeled "LAN0, is needed for powering the device over PoE.
Currently eth0 and eth1 are swapped: Setup interface is on the PoE powered port LAN0", while the WAN port is assigned to "LAN1".
It should be the other way round, otherwise two cables are needed.

**Comment by NeoRaider** [comment]
It is possible that LAN and WAN are switched; the PoE port should be WAN by default, but I have no way to check that without access to the hardware. However, I'm surprised that the setup interface is on LAN - as far as I can tell, the device is correctly marked as an "outdoor device", which should move the setup interface to the WAN port.

Please post the output of `tail /lib/gluon/core/sysconfig/*_ifname`.

**Comment by citronalco** [comment]
```
==> /lib/gluon/core/sysconfig/lan_ifname <==
eth1

==> /lib/gluon/core/sysconfig/setup_ifname <==
eth0

==> /lib/gluon/core/sysconfig/wan_ifname <==
eth0
```

**Comment by NeoRaider** [comment]
You wrote that the setup interface is on the PoE-powered port LAN0. The same port becomes uplink/WAN after configuration, which is the intended setup for outdoor devices.

**Comment by citronalco** [comment]
```
==> /lib/gluon/core/sysconfig/lan_ifname <==
eth1

==> /lib/gluon/core/sysconfig/setup_ifname <==
eth0

==> /lib/gluon/core/sysconfig/wan_ifname <==
eth0

```

Configuration on the PoE powered port. WAN is on the non-powered port.
I'd expect it either the other way round, or that both is on the PoE port.

**Comment by NeoRaider** [comment]
Are you sure? `setup_ifname` and `wan_ifname` are both shown as `eth0`, so they must be the same port.

**Comment by citronalco** [comment]
Rechecked, and you're right: Config is also on the non-powered port. Sorry.

**Comment by NeoRaider** [comment]
Should be fixed by #2385, please test.

**Comment by citronalco** [comment]
Tested, it works: Setup and WAN are now both on the PoE port. Thank you!
